doc/releases/changelog-dev.md

```diff
@@ -4,6 +4,34 @@
 
 <h3>New features since last release</h3>
 
+* A new operation `SpecialUnitary` was added, providing access to an arbitrary
+  unitary gate via a parametrization in the Pauli basis.
+  [(#3650)](https://github.com/PennyLaneAI/pennylane/pull/3650)
+ 
+  The new operation takes a single argument, a one-dimensional `tensor_like`
+  of length `4**num_wires-1`, where `num_wires` is the number of wires the unitary acts on.
+  
+  For example, on a single qubit, we may define
+  
+  ```pycon
+  >>> theta = np.array([0.2, 0.1, -0.5])
+  >>> U = qml.SpecialUnitary(theta, 0)
+  >>> U.matrix()
+  array([[ 0.8537127 -0.47537233j,  0.09507447+0.19014893j],
+         [-0.09507447+0.19014893j,  0.8537127 +0.47537233j]])
+  ```
+  
+  A single non-zero entry in the parameters will create a Pauli rotation:
+
+  ```pycon
+  >>> x = 0.412
+  >>> theta = x * np.array([1, 0, 0]) # The first entry belongs to the Pauli word "X"
+  >>> su = qml.SpecialUnitary(theta, wires=0)
+  >>> rx = qml.RX(-2 * x, 0) # RX introduces a prefactor -0.5 that has to be compensated
+  >>> qml.math.allclose(su.matrix(), rx.matrix())
+  True
```

Comment:
Is it worth mentioning the differentiability of the params (even on hw)?

Reply:
Yes, but only once it is implemented! Good pointer though, I'll include a doc string extension in the corresponding PR!

Reply:
Added a comment in the follow-up PR